WalletWasabi.Tests/UnitTests/SmartBlockProviderTests.cs

```diff
@@ -4,14 +4,16 @@
 using System.Threading.Tasks;
 using Microsoft.Extensions.Caching.Memory;
 using NBitcoin;
-using WalletWasabi.Legal;
 using WalletWasabi.Wallets;
 using Xunit;
 
 namespace WalletWasabi.Tests.UnitTests
 {
 	public class SmartBlockProviderTests
 	{
+		/// <summary>
+		/// Tests <see cref="SmartBlockProvider.GetBlockAsync(uint256, CancellationToken)"/> behavior.
+		/// </summary>
```

Comment:
Comments those doesn't add any meaning to what is already clear from the code are code smells as they need maintenance and if the meaning of the code changes without the comment, the comment will mislead the reader.

This shouldn't block this PR though.

Reply:
When you rename the method, the comment is modified too (at least in MSVS). Also when the method in the comment does not exist a compiler warning will be visible (also MSVS). 

So I think it's not that easy to break this comment.

Reply:
True, but doesn't address my point: this comment doesn't add any meaning that wouldn't be immediately obvious by just looking at the code.